### PR TITLE
5.5.0 updates

### PR DESCRIPTION
**Pull request by shrug**

### CODE REVIEW

**Comment by underscorgan** [comment]
Does it build?

**Comment by shrug** [comment]
Running a build now. I'll look at that rubocop failure.

**Comment by shrug** [comment]
Latest builds are up: https://hub.docker.com/u/puppet/dashboard/

**Comment by underscorgan** [comment]
(for future reviewers, that url didn't work for me, but just removing the /dashboard let me see things, and I saw that puppetserver images had been recently pushed)

**Comment by e-gris** [comment]
I get a 404 on the hub.docker.com page. Are there some permissions settings needed?

File: puppet-agent-alpine/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM alpine:3.4
 
-ENV PUPPET_VERSION="5.4.0" FACTER_VERSION="2.5.1"
+ENV PUPPET_VERSION="5.5.0" FACTER_VERSION="2.5.1"
```
Comment (mwaggett) [inline puppet-agent-alpine/Dockerfile]: why such an old version of facter?

Comment (shrug) [reply puppet-agent-alpine/Dockerfile]: Have to use Ruby facter on alpine, at least until we can get a cfacter build working

Comment (mwaggett) [reply puppet-agent-alpine/Dockerfile]: ahh

**Comment by mwaggett** [comment]
seems reasonable to me but I don't know enough to feel good about merging

**Pull request merged**